vm/systemSmartContracts/delegation_test.go

```diff
@@ -2601,7 +2601,11 @@ func prepareReDelegateRewardsComponents(
 	args.DelegationSCConfig.MaxServiceFee = 10000
 	args.DelegationSCConfig.MinServiceFee = 0
 	enableEpochsHandler, _ := args.EnableEpochsHandler.(*enableEpochsHandlerMock.EnableEpochsHandlerStub)
-	enableEpochsHandler.IsReDelegateBelowMinCheckFlagEnabledField = extraCheckEpoch == 0
+	if extraCheckEpoch == 0 {
```

Comment:
instead of this ugly if...else we can write:
```GO
enableEpochsHandler.IsReDelegateBelowMinCheckFlagEnabledInEpochCalled = func(epoch uint32) bool {
    return extraCheckEpoch == 0
}
```

Reply:
indeed :man_facepalming: